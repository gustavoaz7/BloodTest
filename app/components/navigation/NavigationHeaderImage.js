import React from 'react';
import { View, Image } from 'react-native';
import styled from 'styled-components/native';

const logo = require('../../../assets/logo.png');

export default function NavigationHeaderImage() {
  return (
    <Container>
      <HeaderImage source={logo} resizeMode="contain" resizeMethod="resize" />
    </Container>
  );
}

const Container = styled(View)`
  flex: 1;
`;

export const HeaderImage = styled(Image)`
  flex: 1;
  width: 100%;
`;
