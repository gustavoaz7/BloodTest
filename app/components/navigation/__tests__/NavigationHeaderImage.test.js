import React from 'react';
import { shallow } from 'enzyme';
import NavigationHeaderImage, { HeaderImage } from '../NavigationHeaderImage';

const logo = require('../../../../assets/logo.png');

describe('NavigationHeaderImage', () => {
  it('renders without crashing', () => {
    shallow(<NavigationHeaderImage />);
  });

  it('renders logo', () => {
    const wrapper = shallow(<NavigationHeaderImage />);
    expect(wrapper.find(HeaderImage).props().source).toEqual(logo);
  });

  it('matches snapshot', () => {
    const wrapper = shallow(<NavigationHeaderImage />);
    expect(wrapper).toMatchSnapshot();
  });
});
