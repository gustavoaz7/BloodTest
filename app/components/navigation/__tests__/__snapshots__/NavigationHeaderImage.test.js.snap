// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationHeaderImage matches snapshot 1`] = `
<Styled(View)>
  <Styled(Image)
    resizeMethod="resize"
    resizeMode="contain"
    source={1}
  />
</Styled(View)>
`;
